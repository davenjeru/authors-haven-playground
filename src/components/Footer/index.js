import React from 'react';
import './Footer.css';

const Footer = () => (
  <footer className="bg-dark footer">
    <p className="m-auto text-muted">
      Â© Copyright 2018 Authors Haven - All rights reserved.
    </p>
  </footer>
);

export default Footer;
