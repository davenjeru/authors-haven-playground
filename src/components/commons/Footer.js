import React from 'react';

const Footer = () => (
  <footer
    className="bg-dark text-white"
    style={{
      position: 'fixed',
      bottom: '0',
      width: '100%',
    }}
  >
    <div className="row">
      <div className="col-md-12 mt-3">
        <p className="text-center text-muted">
          Â© Copyright 2018 Authors Haven - All rights reserved.
        </p>
      </div>
    </div>
  </footer>
);

export default Footer;
