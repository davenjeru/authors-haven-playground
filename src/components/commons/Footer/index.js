import React from 'react';
import './Footer.css';

const Footer = () => (
  <footer
    className="bg-dark"
  >
    <div className="text-center">
      <span className="text-muted">
      Â© Copyright 2018 Authors Haven - All rights reserved.
      </span>
    </div>
  </footer>
);

export default Footer;
