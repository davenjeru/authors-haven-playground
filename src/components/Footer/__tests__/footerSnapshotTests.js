import renderer from 'react-test-renderer';
import React from 'react';
import Footer from '..';

describe('The Footer', () => {
  it('should not regress', () => {
    const tree = renderer.create(<Footer />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
