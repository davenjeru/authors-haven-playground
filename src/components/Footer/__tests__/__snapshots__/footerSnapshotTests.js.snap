// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Footer should not regress 1`] = `
<footer
  className="bg-dark footer"
>
  <p
    className="m-auto text-muted"
  >
    Â© Copyright 2018 Authors Haven - All rights reserved.
  </p>
</footer>
`;
